media-libs/harfbuzz/harfbuzz-9999.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{8..10} pypy3 )
```

Comment:
Did you test the package with `python3_10` and `pypy3`? If not, please do not add these.

Reply:
python10 work, pypy3 isn't used by meson